### ISSUE TITLE
Wrong error

### ISSUE THREAD

**Issue by carowright** [raml-org/raml-js-parser-2#742]
I have this RAML

```
#%RAML 0.8
title: 00170156
version: 1.0

/policies: 
  /{id}: 
    uriParameters: 
      id: 
        required: true 
        minLength: 12 
        maxLength: 12 
        type: string 
        example: "123456789-12"
    get:
```
Although the raml is correct, I get the following error: 

```
{
code: "MINMAX_RESTRICTION_VIOLATION"
isWarning: true
message: "'id.minLength=12' i.e. string length should not be less than 12"
...
}
```

**Comment by agustinlg** [comment]
Testing from the browser, using `expand(true)` and getting validation errors from `toJson({rootNodeDetails:true})`

**Comment by dreamflyer** [comment]
will be published in v1.1.23.
